scripts/assets/file_content_validation_checks.textproto

```diff
@@ -3,6 +3,16 @@ file_content_checks {
   prohibited_content_regex: "^import .+?support.+?$"
   failure_message: "AndroidX should be used instead of the support library"
 }
+file_content_checks {
```

Comment:
I was actually thinking we should have two: one that prohibits "announceForAccessibility\(" and suggests using AccessibilityChecker, and a second that checks for "announceForAccessibilityForView\(" with the message you have below. We generally always want people to use the utility, but it still requires approval to avoid over-use.

Reply:
Done, added second check and a test.